Lib/fontTools/ttLib/ttFont.py

```diff
@@ -748,8 +832,12 @@ def getTableData(self, tag):
             raise KeyError(tag)
 
     def getGlyphSet(
-        self, preferCFF=True, location=None, normalized=False, recalcBounds=True
-    ):
+        self,
+        preferCFF: bool = True,
+        location: dict[str, float] | None = None,
```

Comment:
Note that this won't allow you to pass `my_location: dict[str, int]`, but I'm not sure if that's such a bad thing here.

Reply:
One could turn this into `dict[str, float] | dict[str, int] | None`. Not sure if there's a more elegant way.